cypress/support/commands.ts

```diff
@@ -221,11 +225,8 @@ Cypress.Commands.add("preventPrint", () => {
 });
 
 Cypress.Commands.add("closeNotification", () => {
-  cy.get(".pnotify")
-    .should("exist")
-    .each(($div) => {
-      cy.wrap($div).click();
-    });
+  cy.wait(5000); //wait for the notification to disappear
```

Comment:
```suggestion
  cy.wait(5000); //wait for the notification to disappear
```

don't use hardcorded wait, look into another solution

Reply:
> don't use hardcorded wait, look into another solution

I understand this is not the best solution, but I have tried the following methods to drag the notification upwards to close it, unfortunately they did not work

1. using `mousemove`

```tsx
Cypress.Commands.add("closeNotification", () => {
  cy.get("li[data-sonner-toast] div[data-title]")
    .should("exist")
    .trigger('mousedown', { which: 1, force: true }) //start the drag
    .trigger('mousemove', { clientY: -100, force: true }) // drag upward
    .trigger('mouseup', { force: true }); // end the drag
    });

  // ensure the toast is removed from the DOM
  cy.get("li[data-sonner-toast]").should("not.exist");
});
```

1. using `pointermove`
    
    ```tsx
    Cypress.Commands.add("closeNotification", () => {
      cy.get("li[data-sonner-toast]")
        .trigger('pointerdown', { which: 1, force: true }) // start the drag
        .trigger('pointermove', { clientX: 0, clientY: -100, force: true }) // drag upward
        .trigger('pointerup', { force: true }); // end the drag
    
      // ensure the toast is removed from the DOM
      cy.get("li[data-sonner-toast]").should('not.exist');
    });
    ```
    
2. using `toast.dismiss()`
    
    ```tsx
    Cypress.Commands.add("closeNotification", () => {
      cy.window().then((win) => {
        if (win.toast) {
          win.toast.dismiss(); // dismiss all toasts programmatically
        } else {
          throw new Error("Sonner `toast` object not found.");
        }
      });
    
      // ensure the toast is removed from the DOM
      cy.get("li[data-sonner-toast]").should("not.exist");
    });
    ```
    

I am continuing to explore better solutions and would greatly appreciate any advice or suggestions you might have!